### PR TITLE
Add missing toolbox deps webui

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Review by jkonecny12 — COMMENTED** [review]
I'm opposed to this directly but I'm not sure if it is worth the troubles. I see it this way:

Benefit:
You can create smaller toolbx container

Drawback:
It's giving you one more complexity to for setup. One more thing developers have to take into account if they need that or that.


The main question:
Is the benefit worth the drawback in this case? What is the size reduction would be first great question to ask.

**Review by jstodola — APPROVED** [review]
I like the changes, since I run into these missing dependencies as well when experimenting with webUI tests (and then forgot to do something about it).

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

Could be really useful in the future for the likely case of splitting the Web UI code to its own repo eventually.

**Comment by KKoukiou** [comment]
> I'm opposed to this directly but I'm not sure if it is worth the troubles. I see it this way:
> 
> Benefit: You can create smaller toolbx container

I would not say that's so important, but keeping a clean development environment is. People not working on webui should not deal with dependencies like npm, virt-install, qemu and such.

> 
> Drawback: It's giving you one more complexity to for setup. One more thing developers have to take into account if they need that or that.

Luckily contributors don't need to run this scripts on their daily development workflow. It's mostly once if you run these on the host directly, and each time you create a new toolbox, if working on a toolbox. 

> 
> The main question: Is the benefit worth the drawback in this case? What is the size reduction would be first great question to ask.

Size reduction is not my main argument here. Keeping this cleanly seperated, so that non-webui developers can keep their envs clean from webui dependencies is one. And also easiing the posibility for a future split of the repos, if you decide so in the future.

**Review by VladimirSlavik — COMMENTED** [review]
Thank you! Main idea is good. I would like to see commits squashed. Most importantly though, this needs rpm-build back (see below).

File: scripts/testing/install_webui_dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    firefox \
+    chromium-headless \
+    npm
+
+# Additional Web UI tests dependencies - these are specific to anaconda Web UI tests and not listed in cockpit
+# shellcheck disable=SC2068
+dnf install $@ \
+    qemu-img \
+    rpm-build \
+    virt-install
```
[the remark this answers is not in the document]
Comment (jkonecny12) [reply scripts/testing/install_webui_dependencies.sh:27]: You can enable repo on both calls and similar things. It's could act weird but I think it's still better with than without it. The question is if we want to have two DNF calls. It's in general slowing things down.

[the remark this answers is not in the document]
Comment (VladimirSlavik) [reply scripts/testing/install_webui_dependencies.sh]: Just to make it clearer, I'm +1 on making it a single dnf call :)

[the remark this answers is not in the document]
Comment (KKoukiou) [reply scripts/testing/install_webui_dependencies.sh]: It is one dnf call now for webui deps.

**Pull request merged**